lib/berlim/sales/sales.ex

```diff
@@ -26,4 +32,45 @@ defmodule Berlim.Sales do
   def change_plan(plan \\ %Plan{}, plan_attrs \\ %{}) do
     Plan.changeset(plan, plan_attrs)
   end
+
+  def list_orders(params) do
+    Order
+    |> order_by(desc: :inserted_at)
+    |> Repo.paginate(params)
+  end
+
+  def get_order!(id), do: Repo.get!(Order, id)
+
+  def create_order(order_attrs) do
+    %Order{}
+    |> change_order(set_order_attrs(order_attrs))
+    |> Repo.insert()
+  end
+
+  def update_order(order, order_attrs) do
+    order
+    |> change_order(set_order_attrs(order_attrs))
+    |> Repo.update()
+  end
+
+  def delete_order(%Order{} = order) do
+    Repo.delete(order)
+  end
+
+  def change_order(order \\ %Order{}, order_attrs \\ %{}) do
+    Order.changeset(order, order_attrs)
+  end
+
+  defp set_order_attrs(%{"taxi_id" => taxi_id, "monthly_date" => monthly_date} = order_attrs) do
+    taxi = Accounts.get_taxi!(taxi_id)
+    taxi = Repo.preload(taxi, :plan)
+
+    Map.merge(order_attrs, %{
+      "monthly_date" => parse_date(monthly_date),
+      "value" => taxi.plan.value,
+      "status" => "paid",
```

Comment:
não entendi essa order manual, qual não será manual? se o cara passar na máquina de cartão dentro da voo, também será manual, vc n está deixando escolher a forma de pagamento, o status, etc

Reply:
se essa order manual for só para dinheiro, acho que merece um módulo específico para criar orders em dinheiro